source/includes/administration/_resource_commitments.md

```diff
@@ -0,0 +1,343 @@
+## Resource commitments
+
+Resource commitments allow you to set specific commitment levels on cloud resources `(compute, network & storage)` avaialble on a [service-connection](#administration-service-connections). This assures commitment to a specific set of resources on a fixed monthly price. Any additional resource usage will be billed on the utility price defined by the CMC [pricing](#administration-pricing). Commitments can either be restricted to a specific date range or be un-restricted with no end date. Whilst the commitment price is decided by the pre-defined [pricing](#administration-pricing), various discounts can be given on the resource utility price. Resource commitments allow Cloud-MC users to receive services at fixed discounted rates whilst enabling additional usage at utility cost.
```

Comment:
CloudMC instead of Cloud-MC

Reply:
Fixed in 738496526a7aa1f45bc86f95f7c32cc0e4ed3320